cmd/opm/alpha/bundle/build.go

```diff
@@ -55,14 +55,8 @@ func newBundleBuildCmd() *cobra.Command {
 	}
 
 	bundleBuildCmd.Flags().StringVarP(&packageNameArgs, "package", "p", "", "The name of the package that bundle image belongs to")
```

Comment:
IMO we need to update this help message to let users know that it's required if they're not pointing to a directory in the nested bundle format